### PR TITLE
don't treat the no data case differently in the Learner1D

### PR DESCRIPTION
**Pull request by basnijholt**

### CODE REVIEW

File: adaptive/learner/learner1D.py [hunk 1]
```diff
@@ -590,12 +590,10 @@ def plot(self):
             Plot of the evaluated data.
         """
         hv = ensure_holoviews()
-        if not self.data:
-            p = hv.Scatter([]) * hv.Path([])
-        elif not self.vdim > 1:
+        if not self.vdim > 1:
```
Comment (akhmerov) [inline adaptive/learner/learner1D.py]: `if self.vdim <= 1`

Comment (akhmerov) [reply adaptive/learner/learner1D.py]: Ehm, `== 1`, I guess we don't allow `vdim == 0`?

**Comment by akhmerov** [comment]
Other than the comment above LGTM

File: adaptive/learner/learner1D.py (hunk 1 above)
Comment (basnijholt) [reply adaptive/learner/learner1D.py]: I vaguely remember that `vdim` could be 0, but looking at the code I don't see how (and why).

I am a bit hesitant to remove it.

Comment (akhmerov) [reply adaptive/learner/learner1D.py]: Even then `<=` is easier to read than `not >`.

Comment (basnijholt) [reply adaptive/learner/learner1D.py]: Right, I amended the commit.

**Pull request merged**